docs/conf.py

```diff
@@ -66,7 +69,7 @@ def try_import(mod_name):
 project, version, url, author = output_bytes.decode('utf-8').strip().split('\n')
 
 # General information about the project.
-copyright = '2001-2016 ' + author
+copyright = '2001-%d %s' % (datetime.date.today().year, author)
```

Comment:
Ooh. I like this. I'll want to adopt this change in [jaraco/skeleton](/jaraco/skeleton) from where the original code came, but fine to use this here for now.